WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -172,7 +177,8 @@ private extension ItemListView {
         // Note that navigation is handled by the ItemList in iOS 17, so any changes to this should be reflected in ItemListRow.
         switch parentItem {
         case let .variableParentProduct(parentProduct):
-            ChildItemList(parentItem: parentItem, title: parentProduct.name)
+            let itemsStack = selectedItemType == .products ? posModel.itemsViewState.itemsStack : posModel.couponsViewState.itemsStack
```

Comment:
Well, this is specific to items. We could use `posModel.itemsViewState.itemsStack` for `.variableParentProduct`